packages/react-tabs/src/components/Pivot/Pivot.base.tsx

```diff
@@ -121,9 +129,9 @@ export const PivotBase: React.FunctionComponent<IPivotProps> = React.forwardRef(
           key={itemKey}
           className={css(className, isSelected && classNames.linkIsSelected)}
           // eslint-disable-next-line react/jsx-no-bind
-          onClick={onLinkClick.bind(this, itemKey)}
+          onClick={(ev: React.MouseEvent<HTMLElement>) => onLinkClick(itemKey!, ev)}
           // eslint-disable-next-line react/jsx-no-bind
-          onKeyDown={onKeyDown.bind(this, itemKey)}
+          onKeyDown={(ev: React.KeyboardEvent<HTMLElement>) => onKeyDown(itemKey!, ev)}
```

Comment:
Can we move into `consts` above so that we don't have lambdas in the render function?

Reply:
That wouldn't actually change the fact that the function enclosures are newly created on each call to render. We'd need to use `Rect.useCallback` to prevent them from changing... However, using a hook in renderPivotLink isn't technically safe, since the number or order of pivots could change between renders, resulting in the hooks being called in a different order between renders.

We'll need to extract the header links to their own component in order to fix this in the future. It'll likely be a re-composed Button.